<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertGui</class>
 <widget class="QMainWindow" name="ConvertGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConvertGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>187</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <widget class="QPushButton" name="convBt">
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBtn">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Pictures</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>561</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblPicFrom">
         <property name="text">
          <string>From:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbPicFrom">
         <item>
          <property name="text">
           <string>PDF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JPG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PNG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GIF</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="txtPicFrom"/>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnPicFrom">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblPicTo">
         <property name="text">
          <string>To:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cmbPicTo">
         <item>
          <property name="text">
           <string>PDF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JPG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PNG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GIF</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="txtPicTo"/>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnPicTo">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Video</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>23</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
